yt/visualization/tests/test_callbacks.py

```diff
@@ -65,6 +67,9 @@
 #  X ray
 #  X line_integral_convolution
 
+# 2D cylindrical data for callback test
+cyl_2d = "WDMerger_hdf5_chk_1000"
```

Comment:
This needs to be a fully qualified path to the hdf5 file, `"WDMerger_hdf5_chk_1000/WDMerger_hdf5_chk_1000.hdf5"`